assets/js/index.js

```diff
@@ -0,0 +1,39 @@
+const addTaskIcon = document.getElementById("add-task-icon");
+const addTaskInput = document.getElementById("add-task-input");
+const errorMessage = document.getElementById("error-message");
+const backendErrorMessage = document.getElementById("backend-error-message");
+
+async function postTodo(todoText) {
+  const url = "http://localhost:3030/todos";
+
+  try {
+    const response = await fetch(url, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ text: todoText }),
+    });
+    if (response.status !=  200 && response.status != 201) {
```

Comment:
1. Why `!=` and not `!==`?
2. Do you get `200` from server in the case of success?